### PR TITLE
Docs :D

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: lib/fluxter.ex [hunk 1]
```diff
@@ -1,6 +1,171 @@
...
+      the configured Fluxter pool will be prefixed by the value of this
+      option. If `nil`, metrics will not be prefixed. Defaults to `nil`.
+    * `:pool_size` - (integer) the size of the connection pool for the given
+      Fluxter pool. **This option can only be configured on a per-pool basis**;
+      configuring it globally for the `:fluxter` application has no
+      effect. Defaults to `5`.
+
+  """
+
+  @type tag_or_field_name :: atom | iodata
```
Comment (whatyouhide) [inline lib/fluxter.ex]: `String.Chars.t`

File: lib/fluxter.ex [hunk 2]
```diff
@@ -1,6 +1,171 @@
...
+  @doc """
+  Should be the same as `write(name, [], fields)`.
+  """
+  @callback write(name :: iodata, fields :: field_value | fields) :: :ok
+
+  @doc """
+  Should be the same as `measure(name, [], [], fun)`.
+  """
+  @callback measure(name :: iodata, fun :: (() -> res)) ::
+    res when res: any
```
Comment (whatyouhide) [inline lib/fluxter.ex]: `result`

File: lib/fluxter.ex [hunk 3]
```diff
@@ -1,6 +1,171 @@
...
+
+  ## Examples
+
+  Assuming a `MyApp.Fluxter` Fluxter pool exists:
+
+      iex> MyApp.Fluxter.write("cpu_temp", [host: "eu-west"], 68)
+      :ok
+
+  """
+  @callback write(name :: iodata, tags :: tags, fields :: field_value | fields) :: :ok
```
Comment (whatyouhide) [inline lib/fluxter.ex]: Less `::` naming :)

**Comment by whatyouhide** [comment]
@lexmag updated the points we discussed.

**Comment by lexmag** [comment]
:shipit:

**Pull request merged**